crates/bevy_anti_aliasing/src/taa/mod.rs

```diff
@@ -151,7 +148,7 @@ impl Default for TemporalAntiAliasing {
 
 /// Render [`bevy_render::render_graph::Node`] used by temporal anti-aliasing.
 #[derive(Default)]
-pub struct TemporalAntiAliasNode;
+struct TemporalAntiAliasNode;
```

Comment:
Why did you remove the `pub` here?

Reply:
Might've been an accident, it looks like it was meant to be kept public.